src/stream/src/executor/hash_agg.rs

```diff
@@ -134,7 +133,7 @@ struct ExecutionVars<K: HashKey, S: StateStore> {
     stats: ExecutionStats,
 
     /// Cache for [`AggGroup`]s. `HashKey` -> `AggGroup`.
-    agg_group_cache: AggGroupCache<K, S>,
+    agg_group_cache: AggGroupCache<K>,
```

Comment:
I would suggest keep the `Strategy` generic parameter here. Though the really difference happens when calling `build_change`, the strategy is a characteristic of the executor.